test/sql/validation.sql

```diff
@@ -172,6 +172,20 @@ SELECT * FROM diffix.show_labels() WHERE objname LIKE 'public.empty_test_custome
 PREPARE prepared(float) AS SELECT discount, count(*) FROM empty_test_customers WHERE discount = $1 GROUP BY 1;
 EXECUTE prepared(1.0);
 
+-- Allow anonymizing JOINs
+SELECT COUNT(*) FROM test_validation AS c
+  INNER JOIN test_purchases ON c.id = cid;
```

Comment:
Are we certain that all AIDs are collected from both relations?
A test with an `EXPLAIN VERBOSE` will help us identify if all aids are being passed to anon aggs such as low_count.